lintercat/lintercat.com
# Re-arrange project to put medicapp in second

### What does this PR do?
Move medicapp to the second spot.

### How should this be manually tested?
1. Open the app
2. Make sure medicapp is in the second spot and rebecabasto.me is in third

### Any background context you want to provide?
This improves the balance between device types and also includes a component with a proper loading animation.

### What are the relevant issues?
Close #8
